lib/features/sign_in/input_field.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
```

Comment:
better to have the same file name and class name
either build_input_field.dart & BuildInputField or input_field.dart & InputField

I would also put into separate folder called /widgets